ui/pages/confirmations/components/confirm/scroll-to-bottom/scroll-to-bottom.tsx

```diff
@@ -71,6 +80,11 @@ const ScrollToBottom = ({ children }: ContentProps) => {
   }, [currentConfirmation?.id, previousId, ref?.current]);
 
   useEffect(() => {
+    if (isTransactionRedesign) {
+      setIsScrollToBottomCompleted(true);
+      return;
+    }
+
     setIsScrollToBottomCompleted(!isScrollable || hasScrolledToBottom);
   }, [isScrollable, hasScrolledToBottom]);
```

Comment:
Should this hook to have `isTransactionRedesign` dependency?

Reply:
Great catch, [done](https://github.com/MetaMask/metamask-extension/pull/27910/commits/a1a3614aa5e12edc8045f399658ee84726e85de3).